Added known limitations documentation (#22)
CitrineInformatics/taurus-documentation: docs/specification/value-types.md

Most commonly a molecular structure will refer to the entire material,
but multiple molecular structures can be used to define fragments of a material, e.g. monomers of a polymer.

> See [Known Limitations](../../known_limitations).  We plan to (**but do not yet**) support two ways of representing a molecular structure:

Reviewer: ```suggestion
There are two ways to represent a molecular structure:
```

* SMILES string

> * InChI string

Reviewer: ```suggestion
* InChI string

See [Known Limitations](../../known_limitations).  **Support for these Value Types is still pending.**
```

Reviewer: Just trying to be consistent with the disclaimer.
